src/modules/roc_netio/target_uv/roc_netio/iclose_handler.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2019 Roc authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+//! @file roc_netio/target_uv/roc_netio/iclose_handler.h
+//! @brief close handler interface.
+
+#ifndef ROC_NETIO_ICLOSE_HANDLER_H_
+#define ROC_NETIO_ICLOSE_HANDLER_H_
+
+#include "roc_netio/ielement.h"
+
+namespace roc {
+namespace netio {
+
+//! Close handler for network element.
+class ICloseHandler {
+public:
+    virtual ~ICloseHandler();
+
+    //! Asynchronous close for network element.
+    //!
+    //! @remarks
+    //!  - Closed element should not be used.
+    //!  - Should be called from the event loop thread.
+    virtual void close(IElement&) = 0;
```

Comment:
I suggest "closeD()" or "handle_closed()" because "close()" implies that the closing operation should be implemented by ICloseHandler but this is wrong.

Reply:
Fixed